sass/directives/_carousels.scss

```diff
@@ -96,3 +96,97 @@
     padding: 0;
   }
 }
+
+// Product Buy Pages Carousel (Why Post and Why Search)
+@mixin product__carousel__item-wrap {
+  @include carousel-wrap;
+
+  .slick-list {
+    @media only screen and (max-width: $small-screen-max) {
+      background-color: $panel-primary;
+    }
+  }
+}
+
+@mixin product__carousel__item {
+  position: relative;
+  outline: 0;
+
+  @media only screen and (max-width: $small-screen-max) {
+    padding: $base-padding;
+  }
+
+  &:hover,
+  &:active,
+  &:focus {
+    text-decoration: none;
+  }
+
+  h3 {
+    margin-bottom: 0;
+    font-size: $base-font-size;
+  }
+
+  span {
+    color: $paragraph-text-color;
+  }
+}
+
+// Testimonial Carousels
+@mixin testimonial__carousel__item-container {
+  @include display(flex);
+  @include flex-direction(row);
+  @include flex-wrap(nowrap);
+  @include justify-content(center);
+  @include align-items(center);
+
+  @media only screen and (max-width: $medium-screen-max) {
+    @include flex-direction(column);
+    text-align: center;
+
+    img {
+      display: inline;
+    }
+  }
+}
+
+@mixin testimonial__carousel__item {
+  position: relative;
+  outline: 0;
+
+  &:hover,
+  &:active,
+  &:focus {
+    text-decoration: none;
+  }
+
+  span {
+    color: $paragraph-text-color;
+  }
+}
+
+@mixin testimonial__carousel__logo {
+  max-width: 120px;
+
+  @media only screen and (min-width: $medium-screen-max) {
+    padding-right: $base-margin;
+  }
+
+  img {
+    // Safari 9 fixes for preserving image aspect ratio
+    flex: 0 0 auto;
```

Comment:
here use `@include flex(0 0 auto)` (if that doesn't work double check the syntax in the bourbon docs)